README.md

```diff
@@ -25,7 +25,7 @@ I'm open to contributions and comments.
 | type spread operator | [work in progress](https://github.com/Microsoft/TypeScript/pull/13470) | [shipped](https://github.com/facebook/flow/commit/ad443dc92879ae21705d4c61b942ba2f8ad61e4d) >=0.42 |
 | ecosystem flexibility | [work in progress](https://github.com/Microsoft/TypeScript/issues/6508) | no extensions |
 | programmatic hooking | architecture prepared, work in progress | work in progress |
-| documentation and resources | <ul><li>very good docs</li><li>many books</li><li>videos</li><li>e-learning resources</li></ul> | <ul><li>incomplete, often vague docs</li><ul> |
+| documentation and resources | <ul><li>very good docs</li><li>many books</li><li>videos</li><li>e-learning resources</li></ul> | <ul><li>very good docs</li><li>less external learning resources</li><ul> |
```

Comment:
I know it's I wouldn't call docs `very good`. They're still incomplete, since they don't contain many of the magic types listed here:

https://github.com/facebook/flow/blob/ab9bf44c725efd2ed6d7e1e957c5566b6eb6f688/src/typing/type_annotation.ml#L252-L567

Some of those missing types are tracked here: https://github.com/facebook/flow/issues/2464

I would use: `undocumented advanced type features` for Flow and perhaps change TypeScript's `very good docs` (which is subjective) to `all type features documented`.

Reply:
Many of those are experimental features which are intentionally undocumented. I can list much larger features in TypeScript which go undocumented. Lots of syntax isn't even mentioned, just look at the documentation for declarations.

It seems pretty clear you just have a bias against Flow and have no intention of being fair. Which is fine, but you might want to note that at the top of your document

Reply:
I use flow at work by choice (of mine), I really don't see the bias here agains flow. The flow's documents are good for basic type theory understanding but not for usage. I would get stuck lots of time getting understand something I found at TS documents without a problem.

Reply:
@thejameskyle My intention is to make the document as unbiased as possible. I specifically list areas where Flow works a bit better, e.g. with the readonly property or the existential type.

I myself am quite on the verge between both technologies since I use and track the development of them both regularly -- Flow for my day job and TypeScript for my side projects. I used to be more pro-TypeScript, but since @calebmer joined the Flow team I can see the shift, and is going in the right direction. If anything, he's doing a great job improving the PR of the project. I also appreciate the post-0.40 changes in Flow, since they improved a lot, esp. tackled some pain-points, and Caleb promises focus on IDE issues this year (currently the biggest culprit IMO).

However, having worked with both technologies and even tutored team-members in both, my experience with the docs (and developer on-boarding in general) is still unequivocally better with TypeScript than with Flow.

There aren't many important undocumented features in the TS docs (I can't recall any of the top of my head), while there are important Flow features that are left undocumented or documented very roughly. 

This also refers to external resources available online -- Flow is improving in that area, but it'll still take time before it'll reach what TypeScript has.

Also, I do not agree with the concept of "intentionally undocumented features". If *it works* and people use it (and they *will* use it, if it provides value to them), it should be documented -- full stop. Perhaps marked as "experimental/unstable/may change at any time". But documented nonetheless. Otherwise you'll get the following: 

1. people encounter somebody else's code that uses those undocumented features
2. they scratch their heads trying to figure it out
3. they'll google/ask somebody about it
4. might not find anything meaningful
5. potentially waste a lot of time

This is why I cannot say with full confidence that Flow has "very good documentation".

If you perceived any other bias in the comparison, please open an issue (or a PR) and I will try to improve on it.

Reply:
@thejameskyle I have a question, because it's related to the problem with the documentation. How do you use `Symbol.iterator`. I found in some code in some definitions the usage of `@@iterator` but it's not relevant because I use it in a Flow file, so I have no clue. In the documentation there is none.
I'm not saying this so you'll answer me or being sarcastic. I just trying to show you how Flow's docs seems to someone who has no relation to Flow team.